integration-testing/rnode_testing/rnode.py

```diff
@@ -87,8 +87,11 @@ def deploy_contract(self, contract):
     def propose_contract(self):
         return self.exec_run(f'{rnode_binary} propose')
 
-    def show_blocks(self):
-        return self.exec_run(f'{rnode_binary} show-blocks')
+    def show_blocks(self, blocks_num):
+        return self.exec_run(f'{rnode_binary} show-blocks -d{blocks_num}')
```

Comment:
```suggestion
        return self.call_rnode('show-blocks', f'-d{blocks_num}')
```

buys you automatic exit code checking